docs/references/react/use-organization-list.mdx

```diff
@@ -1,3 +1,267 @@
+    return <>Loading</>;
+  }
+
+  return (
+    <>
+      <table>
+        <thead>
+          <tr>
+            <th>Email</th>
+            <th>Org name</th>
+          </tr>
+        </thead>
+
+        <tbody>
+          {userInvitations.data?.map((inv) => (
+            <tr key={inv.id}>
+              <th>{inv.emailAddress}</th>
+              <th>{inv.publicOrganizationData.name}</th>
+            </tr>
+          ))}
+        </tbody>
+      </table>
+
+      <button
+        disabled={!userInvitations.hasPreviousPage}
+        onClick={userInvitations.fetchPrevious}
+      >
+        Prev
+      </button>
+      <button
+        disabled={!userInvitations.hasNextPage}
+        onClick={userInvitations.fetchNext}
+      >
+        Next
+      </button>
+    </>
+  );
+};
+
+export default InvitationsTable;
+```
+
+```tsx copy filename="joined-organizations.tsx"
+import { useOrganizationList } from "@clerk/nextjs";
+import React from "react";
+
+const InvitationsTable = () => {
+  const { isLoaded, userInvitations } = useOrganizationList({
+    userInvitations: {
+      infinite: true,
+      keepPreviousData: true,
+    },
+  });
+
+  if (!isLoaded || userInvitations.isLoading) {
+    return <>Loading</>;
+  }
+
+  return (
+    <>
+      <table>
+        <thead>
+          <tr>
+            <th>Email</th>
+            <th>Org name</th>
+          </tr>
+        </thead>
+
+        <tbody>
+          {userInvitations.data?.map((inv) => (
+            <tr key={inv.id}>
+              <th>{inv.emailAddress}</th>
+              <th>{inv.publicOrganizationData.name}</th>
+            </tr>
+          ))}
+        </tbody>
+      </table>
+
+      <button
+        disabled={!userInvitations.hasPreviousPage}
+        onClick={userInvitations.fetchPrevious}
+      >
+        Prev
+      </button>
+      <button
+        disabled={!userInvitations.hasNextPage}
+        onClick={userInvitations.fetchNext}
+      >
+        Next
+      </button>
+    </>
+  );
+};
+
+export default InvitationsTable;
+```
+</CodeBlockTabs>
+
+## Parameters
+useOrganizationList accepts a single object with the following properties
+| Properties | Description |
+| --- | --- |
+| `userMemberships` | `CommonPaginatedParams` (Optional) |
+| `userInvitations` | `CommonPaginatedParams` (Optional) and status which describes the status of an invitation. Defaults to `pending`. |
+| `userSuggestions` | `CommonPaginatedParams` (Optional) and status which describes the status of a suggestion. Defaults to `pending`. |
+
+### `CommonPaginatedParams`
+
+CommonPaginatedParams can be either `true` or an object with the properties described bellow.
+If it `true` all the defaults values will be used.
+
+| Properties | Description |
+| --- | --- |
+| `initialPage` | A number and can be used to skip the first n-1 pages. For example if initialPage is set to 10, it is will skip the first 9 pages and will fetch the 10th page. Defaults to `1`. |
```

Comment:
```suggestion
| `initialPage` | A number that can be used to skip the first n-1 pages. For example, if `initialPage` is set to 10, it will skip the first 9 pages and will fetch the 10th page. Defaults to `1`. |
```